pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/statics/DBConstants.java

```diff
@@ -35,6 +35,7 @@ public class DBConstants {
     public static final String DDP_ABSTRACTION_ALIAS = "a";
     public static final String DRUG_ALIAS = "d";
     public static final String ALIAS_DELIMITER = ".";
+    public static final String DDP_INSTANCE_ID_CAMEL_CASE = "ddpInstanceId";
```

Comment:
Is this truly a database constant?  Or do we have an ES constant class that we can put this into?

Reply:
I will move it to another place with one of my other PRs